test/Microsoft.AspNetCore.Razor.Evolution.Test/CaseSensitiveTagMatchingRuleComparer.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using Microsoft.AspNetCore.Razor.Evolution.Legacy;
+using Microsoft.Extensions.Internal;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Razor.Evolution
+{
+    internal class CaseSensitiveTagMatchingRuleComparer : TagMatchingRuleComparer
```

Comment:
Maybe you'll have to explain to me a few things, but what's the purpose of this?